src/multihash_impl.rs

```diff
@@ -69,6 +69,10 @@ pub enum Code {
     #[cfg(feature = "blake2s")]
     #[mh(code = 0xb260, hasher = crate::Blake2s256, digest = crate::Blake2sDigest<crate::U32>)]
     Blake2s256,
+    /// BLAKE3 (32-byte hash size)
+    #[cfg(feature = "blake3")]
+    #[mh(code = 0x1e, hasher = crate::Blake3, digest = crate::Blake3Digest<crate::U64>)]
```

Comment:
You are saying "32-byte hash size", but you are using `crate::U64`, which is 64-byte.

Reply:
Thanks for mentioning, I forgot to change after copy pasta.

Reply:
multihash_impl breaks when splitting Blake3 to multiple sizes, as the multihash table only defines one Blake3 code with the definition being ["BLAKE3 has a default 32 byte output length. The maximum length is (2^64)-1 bytes."](https://github.com/multiformats/multicodec/blob/master/table.csv#L21) we could roll with just Blake3_256 for now and then later add support extendable output. Or make a PR to the table to add multiple size definitions.